models/tutorial/index.js

```diff
@@ -82,3 +104,19 @@ function handleClockEvent (events, event, send) {
     })
   }
 }
+
+function getLevelIfExistent (category, index) {
```

Comment:
I know we only have 1 more day left, but I'm not sure if we should leave this function like that